src/docs/components/code-block.svelte

```diff
@@ -15,29 +15,41 @@
 	};
 </script>
 
-<div use:setCodeString {...$$restProps} data-rehype-pretty-code-fragment use:preTabIndex>
+<div
+	use:setCodeString
+	class={cn($$restProps.class, 'force-dark')}
+	{...$$restProps}
+	data-rehype-pretty-code-fragment
+	use:preTabIndex
+>
 	<slot />
 </div>
 <button
-	class={cn('absolute right-4 top-12 z-10', copyBtnClasses)}
+	class={cn('force-dark absolute right-4 top-12 z-10 text-white', copyBtnClasses)}
```

Comment:
Would be nice to see the code blocks follow the theme too, but this is a great start :)